docs/evaluate/development-features/schedules.mdx

```diff
@@ -0,0 +1,39 @@
+---
+id: schedules
+title: Schedules - Temporal Development Feature
+description: Discover how to automate and control your Workflow executions with Temporal's scheduling capabilities. Learn the benefits of scheduling Workflows and explore best practices to ensure timely and efficient execution of your business processes.
+sidebar_label: Schedules
+tags:
+  - schedules
+  - Workflow-automation
+  - temporal-Workflows
+  - scheduling
+keywords:
+  - temporal schedules
+  - automate Workflows
+  - schedule Workflow executions
+  - temporal scheduling best practices
+  - Workflow automation temporal
+  - schedule management
+  - temporal Workflow scheduling
+---
+
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
+Schedules in Temporal allow you to control the timing of your Workflow executions, enabling you to handle time-sensitive tasks with precision.
+By scheduling Workflows, you can automate repetitive tasks, reduce the need for manual intervention, and ensure the timely execution of your business processes.
+
+Temporal's scheduling capabilities provide a robust framework for managing Workflow executions.
+With schedules, you can create, backfill, delete, describe, list, pause, trigger, and update Workflow executions, giving you complete control over your automation processes.
+Whether you need to schedule a Workflow to run at regular intervals or at a specific future time, Temporal makes it easy to ensure your Workflows are executed exactly when needed.
```

Comment:
I think this is redundant because it's similar to other stuff earlier on in the page personally!! But if you decide to keep it:

```suggestion
Whether you need to schedule a Workflow Execution to run at regular intervals or at a specific future time, Temporal makes it easy to ensure your Workflow Executions are executed exactly when needed.
```